Cargo.toml

```diff
@@ -48,8 +60,10 @@ time = { version = "0.3", features = ["wasm-bindgen"] }
 criterion = { version = "0.4", default-features = false }
 
 [features]
-default = ["use_pem"]
+default = ["use_pem", "aws_lc_rs"]
```

Comment:
should we have a default crypto backend or let users pick the one they want?

Reply:
I mean, that's up to you. I personally think it's nice to have a reasonable default.